examples/platform/linux/AppMain.cpp

```diff
@@ -369,6 +369,71 @@ class SampleTestEventTriggerHandler : public TestEventTriggerHandler
     }
 };
 
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+void int_array_add_unique(uint16_t ** res, uint16_t a)
```

Comment:
Add a comment on what this method does and usage examples.

Reply:
I will change it